lib/Minz/ModelPdo.php

```diff
@@ -136,14 +141,13 @@ public function __construct(?string $currentUser = null, ?Minz_Pdo $currentPdo =
 				$ex = $e;
 			}
 			sleep(2);
-		}
+			$db = Minz_Configuration::get('system')->db;
 
-		$db = Minz_Configuration::get('system')->db;
-
-		throw new Minz_PDOConnectionException(
+			throw new Minz_PDOConnectionException(
 				$ex->getMessage(),
 				$db['user'], Minz_Exception::ERROR
 			);
+		}
```

Comment:
What happened here? This looks quite wrong

Reply:
if I don't do this phpstan returns an error because $ex is unknown at this point

Reply:
I have reverted without problem, as far as I can see. But the point was that the logic was broken with your changes.